src/constants/routes.js

```diff
@@ -3,3 +3,4 @@ export const EPISODES = "/episodes";
 export const EPISODE = "/episode";
 export const LOCATION = "/location";
 export const CHARACTER = "/character";
+export const URL = "https://rickandmortyapi.com/api";
```

Comment:
Instead of using the URL const, you could have used the following function:

`function makeApi() {
  return axios.create({
    baseURL: "https://rickandmortyapi.com/api/",
  });
}`

This function creates a base URL that can be called as a parameter in other functions that require this base URL plus the rest of the URL. That way you could save time. Another think to take into consideration is using a function such as the following one to make your requests to the endpoints:

`export function getEpisodes(page = 1, api = makeApi()) {
  return api.get(`/episode?page=${page}`);
}`